.pre-commit-config.yaml

```diff
@@ -1,8 +1,9 @@
 repos:
   - repo: https://github.com/ambv/black
-    rev: 20.8b1
+    rev: '20.8b1'
     hooks:
       - id: black
+        language_version: python3.8
```

Comment:
Are you sure we want to keep `language_version` in here? I'm not completely sure, but I think that it will break the pre-commit config for anyone *not* on 3.8. Could be wrong though.

Reply:
Hmmm, that might be true. I'll remove it just to be sure. Thanks.